src/ploneintranet/workspace/browser/templates/content_macros.pt

```diff
@@ -66,12 +111,12 @@
 
                     </label>
 
-                <label class="attachments">Attached documents from this workspace
+                <label class="attachments"  tal:define="read_only python:False">Attached documents from this workspace
                     <input tal:attributes="disabled read_only" class="pat-autosuggest documents" data-pat-autosuggest="" type="text" value="">
```

Comment:
Not consciously. I'd probably like to blame that on a merge mistake. We don't need or use it, so we can disable it again. @cillianderoiste agree?